src/components/Button/tests/Button.test.tsx

```diff
@@ -133,6 +133,13 @@ describe('<Button />', () => {
     });
   });
 
+  describe('iconPlacement', () => {
+    it('renders the icon before by default', () => {
+      const button = mountWithAppProvider(<Button icon={PlusMinor}>Content</Button>);
+      console.log(button);
```

Comment:
🔥 This may be a bit tricky to test. I've never had to directly test the order in which children render before

Bit of pseudo code but you might be able to pull `children[0]`and expect it to contain react text and `children[1]` to contain react component Icon. Not 100% sure that will work off the top of my head though.

Reply:
Yea I was thinking in that direction too, but was hoping to find a more elegant pattern to use.